lib/clients/BaseClient.py

```diff
@@ -531,6 +531,47 @@ def decrypt_and_extract_tarball_of_directory(self, encrypted_tarball_name, direc
             return result
         return None
 
+    def create_tarball_of_directory(self, directory_to_tar, tarball_name):
+        """Create a tarball of a directory.
+
+        :param directory_to_tar: the path to the directory to be archived
+        :param tarball_name: the path where to store the resulting archive
+
+        :Example:
+            ::
+                iaas_client.create_tarball_of_directory('/tmp/backup/files.tar.gz', '/var/vcap/store/blueprint/files')
+        """
```

Comment:
Just reverse the order in the example. As it is different from actual arguments' order.